orca-core/src/test/groovy/com/netflix/spinnaker/orca/pipeline/model/TestPipelineStage.groovy

```diff
@@ -0,0 +1,34 @@
+package com.netflix.spinnaker.orca.pipeline.model
+
+import com.netflix.spinnaker.orca.pipeline.StageDefinitionBuilder
+import com.netflix.spinnaker.orca.pipeline.TaskNode
+import groovy.transform.CompileStatic
+import groovy.transform.ToString
+
+@CompileStatic
+@ToString(includeNames = true, includeSuper = true)
+class TestPipelineStage extends PipelineStage implements StageDefinitionBuilder {
```

Comment:
Yes, it is identical in content, but it only implements `StageDefinitionBuilder` and I need something that also extends `AbstractStage` or I don't have `context`, amongst other things.  Open to better alternatives here.